src/init.cpp

```diff
@@ -197,120 +207,352 @@ static void CreateNewConfigFile()
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-blockprioritysize=<n>", "Set maximum size of high-priority/low-fee transactions in bytes"
+                                             " (default: 27000)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-snapshotdownload", "Download and apply latest snapshot",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-snapshoturl=<url>", "Optional: URL for the snapshot.zip file (ex: "
+                                         "https://sub.domain.com/location/snapshot.zip)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-snapshotsha256url=<url>", "Optional: URL for the snapshot.sha256 file (ex: "
+                                               "https://sub.domain.com/location/snapshot.sha256)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-disableupdatecheck", "Optional: Disable update checks by wallet",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-updatecheckinterval=<n>", "Optional: Check for updates every <n> hours (default: 120, minimum: 1)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-updatecheckurl=<url>", "Optional: URL for the update version checks (ex: "
+                                            "https://sub.domain.com/location/latest",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-resetblockchaindata", "Reset blockchain data. This argument will remove all previous blockchain data",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-loadblock=<file>", "Imports blocks from external file on startup",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    //TODO: Implement reindex option
+    //argsman.AddArg("-reindex", "Rebuild chain state and block index from the blk*.dat files on disk",
+    //               ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-settings=<file>", strprintf("Specify path to dynamic settings data file. Can be disabled with"
+                                                 " -nosettings. File is written at runtime and not meant to be edited by"
+                                                 " users (use %s instead for custom settings). Relative paths will be"
+                                                 " prefixed by datadir location. (default: %s)",
+                                                 GRIDCOIN_CONF_FILENAME, GRIDCOIN_SETTINGS_FILENAME),
+                   ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-startupnotify=<cmd>", "Execute command on startup.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
```suggestion
    //argsman.AddArg("-startupnotify=<cmd>", "Execute command on startup.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```
Not implemented yet.